Dockerfile

```diff
@@ -4,9 +4,17 @@ MAINTAINER Code Climate <hello@codeclimate.com>
 WORKDIR /usr/src/app
 COPY package.json /usr/src/app/
 
-RUN apk --update add git && \
+RUN apk --update add git jq && \
     npm install && \
-    apk del --purge git
+    git clone https://github.com/eslint/eslint.git && \
```

Comment:
I think this could be a shallow clone, approx 1/10 the download:
```shell
$ git clone https://github.com/eslint/eslint.git 
Cloning into 'eslint'...
remote: Counting objects: 33497, done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 33497 (delta 4), reused 0 (delta 0), pack-reused 33481
Receiving objects: 100% (33497/33497), 9.22 MiB | 928.00 KiB/s, done.
Resolving deltas: 100% (24432/24432), done.
Checking connectivity... done.

netweb@MacBook ~/dev/tmp
$ git clone --depth=1 https://github.com/eslint/eslint.git es2
Cloning into 'es2'...
remote: Counting objects: 1286, done.
remote: Compressing objects: 100% (974/974), done.
remote: Total 1286 (delta 255), reused 591 (delta 174), pack-reused 0
Receiving objects: 100% (1286/1286), 1.34 MiB | 324.00 KiB/s, done.
Resolving deltas: 100% (255/255), done.
Checking connectivity... done.

```